terraso_backend/apps/soil_id/migrations/0016_alter_soildata_slope_steepness_degree_and_more.py

```diff
@@ -0,0 +1,53 @@
+# Copyright © 2024 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+# Generated by Django 5.0.7 on 2024-07-25 20:24
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Is this safe to run on a live database? What will the effect be if existing records violate this constraint?

Reply:
We don't have any custom depth intervals outside this range in the database.